frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/components/search/PropertySearchForm.js

```
@@ -1,102 +1,296 @@
-                ...validation,
-              })}
-              type="number"
-              componentInFront={<div className="employee-card-input employee-card-input--front">+91</div>}
-              //maxlength={10}
-        />
+    <SearchForm onSubmit={onSubmit} handleSubmit={handleSubmit} style={containerStyle} className="pt-property-search">
+      <div style={{ width: "24%" }}>
+        <h3 style={dtat}>Cash Desk</h3>
+
+        <div>
+          <label style={labelStyle}>Assessment Year</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)